examples/interaction-commands.hs

```diff
@@ -49,7 +49,7 @@ interactionCommandExample = do
           discordOnEnd = liftIO $ putStrLn "Ended",
           discordOnEvent = eventHandler,
           discordOnLog = \s -> TIO.putStrLn s >> TIO.putStrLn "",
-          discordGatewayIntent = def {gatewayIntentMembers = True, gatewayIntentPrecenses = True}
+          discordGatewayIntent = def {gatewayIntentMembers = True, gatewayIntentPresences = True}
```

Comment:
Can you also update line 38 `-- | Replies "pong" to every message that starts with "ping"` with something more accurate? Thanks!